public/controllers/agent/components/register-agent.js

```diff
@@ -757,31 +799,31 @@ export const RegisterAgent = withErrorBoundary(
         zIndex: '100',
       };
       const customTexts = {
```

Comment:
Maybe you can try to extract some definitions to an external file to avoid making a huge component. It would greatly improve code readability.

Reply:
Yes, that's right. We will open a new issue to improve the readability of this component.